var/spack/repos/builtin/packages/py-wurlitzer/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyWurlitzer(PythonPackage):
+    """Capture C-level stdout/stderr pipes in Python via os.dup2."""
+
+    homepage = "https://github.com/minrk/wurlitzer"
+    url      = "https://github.com/minrk/wurlitzer/archive/refs/tags/3.0.2.tar.gz"
```

Comment:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Reply:
Thanks for the heads up. This is my first python package. (I think?)

Reply:
First of many? 😄

Reply:
We shall see! You and others have been quite prolific already 😃

Reply:
But if there are any new ones I discover, I will add them. I recently hit a nasty conflict between anaconda and my spack toolchain, so I'm going to try switching for good.